src/compounds/ActionsGroup/ActionsGroup.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { ListGroup } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import SendMessage from './actions/SendMessage'
+import './actions-group.scss'
+
+const ActionsGroup = ({ handleSendingMessages }) => {
+  const [show, setShow] = useState(false)
+  const [action, setAction] = useState(null)
+
+  const handleShow = (componentName) => {
+    setAction(componentName)
+    setShow(true)
+  }
+
+  const handleClose = () => {
+    setAction(null)
+    setShow(false)
+  }
+
+  return (
+    <>
+      <ListGroup className='actions-group'>
+        <ListGroup.Item action variant='primary' onClick={() => handleShow('SendMessage')}>
```

Comment:
the variant listed will determine the background color. each supplier app will have their own brand related primary color that this will refer to. since we are using the color "keywords", it means the actions list will only correspond to those keywords opposed to being able to pass in any color.

it also means that this action will always be the primary color, they can't decide want secondary or something else instead. 

at least not in the mvp.